pkg/operator/operator.go

```diff
@@ -291,22 +292,24 @@ func (o *Operator) sync(key string) error {
 	config := o.Config(key)
 	config.SetImages(o.images)
 
-	factory := manifests.NewFactory(o.namespace, config)
+	factory := manifests.NewFactory(o.namespace, o.namespaceUserworkload, config)
 
 	tl := tasks.NewTaskRunner(
 		o.client,
 		[]*tasks.TaskSpec{
-			tasks.NewTaskSpec("Updating Prometheus Operator", tasks.NewPrometheusOperatorTask(o.client, factory)),
-			tasks.NewTaskSpec("Updating Cluster Monitoring Operator", tasks.NewClusterMonitoringOperatorTask(o.client, factory)),
-			tasks.NewTaskSpec("Updating Grafana", tasks.NewGrafanaTask(o.client, factory)),
-			tasks.NewTaskSpec("Updating Prometheus-k8s", tasks.NewPrometheusTask(o.client, factory, config)),
-			tasks.NewTaskSpec("Updating Alertmanager", tasks.NewAlertmanagerTask(o.client, factory)),
-			tasks.NewTaskSpec("Updating node-exporter", tasks.NewNodeExporterTask(o.client, factory)),
-			tasks.NewTaskSpec("Updating kube-state-metrics", tasks.NewKubeStateMetricsTask(o.client, factory)),
-			tasks.NewTaskSpec("Updating openshift-state-metrics", tasks.NewOpenShiftStateMetricsTask(o.client, factory)),
-			tasks.NewTaskSpec("Updating prometheus-adapter", tasks.NewPrometheusAdapterTaks(o.namespace, o.client, factory)),
-			tasks.NewTaskSpec("Updating Telemeter client", tasks.NewTelemeterClientTask(o.client, factory, config.TelemeterClientConfig)),
-			tasks.NewTaskSpec("Updating configuration sharing", tasks.NewConfigSharingTask(o.client, factory)),
+			//tasks.NewTaskSpec("Updating Prometheus Operator", tasks.NewPrometheusOperatorTask(o.client, factory)),
+			tasks.NewTaskSpec("Updating userworkload Prometheus Operator", tasks.NewPrometheusOperatorUserworkloadTask(o.client, factory, config.UserworkloadConfig)),
+			//tasks.NewTaskSpec("Updating Cluster Monitoring Operator", tasks.NewClusterMonitoringOperatorTask(o.client, factory)),
+			//tasks.NewTaskSpec("Updating Grafana", tasks.NewGrafanaTask(o.client, factory)),
+			//tasks.NewTaskSpec("Updating Prometheus-k8s", tasks.NewPrometheusTask(o.client, factory, config)),
+			tasks.NewTaskSpec("Updating Prometheus-userworkload", tasks.NewPrometheusUserworkloadTask(o.client, factory, config.UserworkloadConfig)),
+			//tasks.NewTaskSpec("Updating Alertmanager", tasks.NewAlertmanagerTask(o.client, factory)),
```

Comment:
I assume these will be uncommented again? :)

Reply:
ooops, indeed :)